force-app/main/default/classes/RecipeTriggerHandler.cls

```diff
@@ -0,0 +1,79 @@
+public class RecipeTriggerHandler{
+    //create method intended to determine if the recipe has any crucial data points missing
+    //if true, mark the recipe as DRAFT
+    //if false, set DRAFT to FALSE 
+    public static void isRecipeDraft(List<Recipe__c> recipeList){
+        system.debug('Begin Recipe Draft Status Evaluation');
+        //loop through each recipe 
+        for(Recipe__c recipe : recipeList ){
+            //evaluate the field values of Name, Active Time, Description, Active Time Units and Servings
+            //if any of these fields are empty or null, set the field DRAFT to TRUE 
+            //if all fields are populated, set the field DRAFT to FALSE 
+            if(recipe.Name == '' || recipe.Active_Time__c == NULL || recipe.Description__c == '' 
```

Comment:
Might suggest using String.isBlank() instead of comparing to '' just to avoid any weird one offs with null values